### ISSUE TITLE
Semi-transparent tintColor puts Pano above content inside it

### ISSUE THREAD

**Issue by panta82** [facebook/react-vr#357]
### Description

When you add a semi-transparent `tintColor` to a `Pano`, that puts it "above" content inside it. Observed on `Text` and `Image` elements. `Box` seems to render correctly. `zIndex` and `elevation` have no effect.

The style I placed:
```javascript
{tintColor: 'rgba(255,255,255,0.7)'}
```

How it looks:

![image](https://user-images.githubusercontent.com/4291324/30956996-31a18b8a-a439-11e7-990c-b93c1687e6f3.png)

I noticed this when I tried to layer multiple panos one inside another. Content was affected by tint color of the "inner" pano.

#### Expected behavior

Pano should be semi-transparent, showing some of the background behind it (eg. a different Pano). Objects "inside" pano should be unaffected.

#### Actual behavior

Pano is drawn over some of the elements.

### Reproduction

```javascript
import React from 'react';
import {
  AppRegistry,
  asset,
  Pano,
  Text,
  View,
} from 'react-vr';

export default class tint_demo extends React.Component {
  render() {
    return (
      <View>
        <Pano source={asset('chess-world.jpg')} style={{tintColor: 'rgba(255,255,255,0.7)'}}/>
        <Text
          style={{
            backgroundColor: '#777879',
            fontSize: 0.8,
            fontWeight: '400',
            layoutOrigin: [0.5, 0.5],
            paddingLeft: 0.2,
            paddingRight: 0.2,
            textAlign: 'center',
            textAlignVertical: 'center',
            transform: [{translate: [0, 0, -3]}],
          }}>
          hello
        </Text>
      </View>
    );
  }
};

AppRegistry.registerComponent('tint_demo', () => tint_demo);
```

### Solution

Sorry, no idea. :/

### Additional Information

* 'react-vr' package version: `1.4.0`
* 'react-vr-web' package version:  `1.4.0`
* Operating System: Linux
* Graphics Card: Integrated,  Intel
* Browser: Chrome
* VR Device: Out of VR

**Comment by andrewimm** [comment]
Should be fixed by 4f288abfcc1be4a98bd798a8e92bdeb1aeb6e1a4, which will be release in 2.0.0 later this week.

**Comment by panta82** [comment]
@andrewimm Sweet!
